### PR TITLE
feat(block): Migrate block

### PR DESCRIPTION
**Pull request by victorscania**
**Describe pull-request**  
fix: migrated block to new storybook

**Solving issue**  
Fixes: [AB#2392](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2392)

**How to test**  
1. Check the component in the new storybook
2. Check if its migrated in accordance with the migrate-checklist

### CODE REVIEW

File: components/src/components/banner/banner.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 :host,
 :root {
-  --sdds-banner-bg-default: var(--sdds-grey-50);
+  --sdds-banner-bg-default: var(red);
```
Comment (mJarsater) [inline components/src/components/banner/banner.scss:3]: Should this be changed back to --sdds-grey-50?

Comment (victorscania) [reply components/src/components/banner/banner.scss]: yeah, my bad. sorted now

File: components/src/components/block/block.stories.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      ],
+    },
+  },
+  argTypes: {
+    type: {
+      name: 'Type',
+      description: 'Pick variant',
+      defaultValue: 'default',
+      control: {
+        type: 'select',
```
Comment (theJohnnyMe) [inline components/src/components/block/block.stories.ts:18]: Please make this an "radio" rather

File: components/src/components/block/block.stories.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    </div>
+    `;
+
+export const Default = Template.bind({});
+Default.args = {};
+Default.parameters = {
+  backgrounds: { default: 'white' },
+};
+
+export const Variant = Template.bind({});
```
Comment (theJohnnyMe) [inline components/src/components/block/block.stories.ts:46]: You can remove this story

File: components/src/components/block/block.stories.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
+export default {
+  title: 'Components/Block/Native',
```
Comment (theJohnnyMe) [inline components/src/components/block/block.stories.ts:2]: Please remove "/Native", we will come back with some better approach there

File: components/src/components/block/block.stories.ts (hunk 3 above)
Comment (theJohnnyMe) [reply components/src/components/block/block.stories.ts:46]: No, it is not important. Controls are sufficient in this one so people can switch and try it out.

Comment (victorscania) [reply components/src/components/block/block.stories.ts]: Sorted

**Pull request merged**